tests/cases/fourslash/codeFixPromiseToAsyncResRef_NoReturnVal.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path='fourslash.ts' />
+
+////function f():Promise<any> {
+////    return fetch('http://yahoo.com').then(res);

```

Comment:
Just avoid potential trademark issues altogether and find/replace these with `typescriptlang.org`